datadog-monitors-monitor-handler/requirements.txt

```diff
@@ -0,0 +1,2 @@
+cloudformation-cli-python-lib==2.1.1
```

Comment:
Should we just have this be required in the common package?

Reply:
Probably yeah, let's do in a next PR, once it's done in the common package ?